Objects/cellobject.c

```diff
@@ -140,8 +140,19 @@ cell_get_contents(PyCellObject *op, void *closure)
     return op->ob_ref;
 }
 
+int
+cell_set_contents(PyCellObject *op, PyCellObject *obj)
+{
+    Py_INCREF(obj);
+    PyCellObject *tmp_op = (PyCellObject *)(op->ob_ref);
```

Comment:
@serhiy-storchaka is right, the correct syntax is 
    `Py_XSETREF(op->ob_ref, (PyObject *)obj);`

But after looking at it more I obj should be a PyObject anyway and not a PyCellObject. I am new at C though so I could be wrong!